src/main.rs

```diff
@@ -54,9 +64,13 @@ fn main() {
         .init()
         .unwrap();
 
+    let leakage: Box<dyn LeakageModel> = match args.leakage_model {
```

Comment:
@LG3696: Is das der beste Weg? Von `<L: LeakageModel>` auf `&dyn LeakageModel>` zu wechseln??

Reply:
Je nachdem :P
Du könntest auch vor dem match beide Modelle anlegen und dann im jeweiligen match Arm eine Referenz auf das Modell rausgeben. Vorteil: kein alloc, Nachteil: Beide Modelle liegen im Speicher